src/lib/repositories/impl_no_sql/order_status_history_repository_impl.py

```diff
@@ -0,0 +1,101 @@
+from datetime import datetime
+
+from src.constants.audit import Status, InternalUsers
+from src.core.mongo_engine_config import mongo_engine_connection
+from src.lib.entities.mongo_engine_odm_mapping import OrderStatusHistory
+from src.lib.repositories.order_status_history_repository import (
+    OrderStatusHistoryRepository,
+)
+
+
+class OrderStatusHistoryRepositoryImpl(OrderStatusHistoryRepository):
+    def __init__(self):
+        mongo_engine_connection()
```

Comment:
Why don't you add this in the ioc instance?

Reply:
done.